contentcuration/contentcuration/frontend/shared/utils/validation.js

```diff
@@ -276,7 +276,10 @@ export function sanitizeAssessmentItemAnswers(answers, removeEmpty = false) {
   }
 
   let sanitizedAnswers = answers.map(answer => {
-    const answerText = answer.answer ? answer.answer.trim() : '';
+    let answerText = answer.answer;
+    if (typeof answerText !== 'number') {
```

Comment:
Perhaps test if it is a string instead? Since the block would then expect that? May also need to default to:
```
let answerText = answer.answer || '';
```